app/screens/staking-unbonding-delegation-amount-input-screen/staking-unbonding-delegation-amount-input-screen.tsx

```diff
@@ -47,9 +47,9 @@ export class StakingUnbondingDelegationAmountInputScreen extends React.Component
     try {
       const { address } = this.props.chain.wallet
       await this.props.txStore.createUnbondingDelegateTx(address)
-      const { target, totalAmount } = this.props.txStore
+      const { target, amount } = this.props.txStore
       const delegatedAmount = this.props.chain.wallet.getDelegation(target).balance
-      if (totalAmount.isGreaterThan(delegatedAmount)) {
+      if (amount.isGreaterThan(delegatedAmount)) {
         throw new Error("UNSTAKE_NOT_ENOUGH_FEE")
```

Comment:
Why is the error `UNSTAKE_NOT_ENOUGH_FEE` instead of `UNDELEGATE_AMOUNT_EXCEED_MAX `?

Reply:
Checked the code, should guard not enough fee here